app/pods/agendas/loading/template.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="vlc-u-box-model-maximize-height vlc-scroll-wrapper vl-u-no-overflow vl-loader-container--not-supported">
+  <p>
```

Comment:
Ik neem aan dat dit een copy paste is van bestaande loaders.

die not-supported css valt mij op, lijkt me custom css voor de "not-supported" pagina, maar staat rond page loader.
Die 6 herhaalde lijnen, kunnen we die in iets herbruikbaar steken ipv te dupliceren
{{web-components/vl-page-loader}}.
Kan ook vervolg cleanup story op de backlog worden.
Either way is fine

Reply:
Terecht. Ik heb de duplicate code van de andere loading pages overgenomen.
Kan hier of kan apart